src/stores/tasks.ts

```diff
@@ -0,0 +1,27 @@
+import { create } from "zustand";
+import { Task, TasksStore } from "../types/Task";
+
+export const useTasksStore = create<TasksStore>((set) => ({
```

Comment:
Zustand has a middleware called persist that you can use to avoid doing all the JSON.parse localStorage... stuff
https://arc.net/l/quote/cbgffbxe

Reply:
I'll check it out